testing/ostinato/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Maintainer: Corentin Henry <corentinhenry@gmail.com>
+pkgname=ostinato
+pkgver=0.7.1
+pkgrel=0
+pkgdesc="Packet/Traffic Generator and Analyzer"
+url="https://www.ostinato.org/"
+arch="all"
+license="GPLv3"
+depends=
+makedepends="qt-dev protobuf-dev libpcap-dev paxctl"
+subpackages="$pkgname-drone $pkgname-gui"
+source="http://dl.bintray.com/pstavirs/ostinato/ostinato-src-$pkgver.tar.gz"
+
+builddir="$srcdir"/$pkgname-$pkgver
+
+build() {
+	cd "$builddir"
+	qmake PREFIX=/usr || return 1
+	make
+}
+
+package() {
+	cd "$builddir"
+	make INSTALL_ROOT=$pkgdir install || return 1
+	paxctl -c -m $pkgdir/usr/bin/drone || return 1
+	paxctl -c -m $pkgdir/usr/bin/ostinato || return 1
```

Comment:
I’ve finally looked how PAX should be handled in abuilds, and this is not the correct way. Look e.g. [here](https://github.com/alpinelinux/aports/blob/master/main/uwsgi/APKBUILD#L72) for inspiration – it uses `paxmark` instead of `paxctl`.